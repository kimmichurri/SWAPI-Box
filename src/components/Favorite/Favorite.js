import React, { Component } from 'react';

export default class Favorite extends Component {
    constructor() {
        super();
    }
    render() {
        //declare a counter here for favorite and interpolate below
        return (
            <button className="favorite-button">view favorites</button>
        )
    }
}