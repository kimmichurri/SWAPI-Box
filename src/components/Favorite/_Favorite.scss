.favorite-button {
    background-color: $dark-blue;
    border-radius: 4px;
    color: #FFF;
    display: flex;
    font-family: $main-font;
    justify-content: center;
    margin: 20px;
    padding: 10px;
    width: 100px;
}