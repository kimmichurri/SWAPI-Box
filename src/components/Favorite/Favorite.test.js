import React from 'react';
import { shallow } from 'enzyme';
import Favorite from './Favorite';

describe('Favorite', () => {
  let wrapper;
  const mockProps = {
    favorites: 4
  }

  beforeEach(() => {
    wrapper = shallow(<Favorite props={mockProps}/>)
  })

  it ('should match the snapshot', () => {
    expect(wrapper.debug()).toMatchSnapshot()
  })
})