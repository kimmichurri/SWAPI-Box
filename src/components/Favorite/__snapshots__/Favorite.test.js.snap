// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Favorite should match the snapshot 1`] = `
"<button className=\\"favorite-button\\">
  view favorites 
</button>"
`;
